components/identity-core/org.wso2.carbon.identity.core/pom.xml

```diff
@@ -178,6 +178,7 @@
                             org.wso2.carbon.user.api; version="${carbon.user.api.imp.pkg.version.range}",
 
                             org.xml.sax,
+                            org.wso2.carbon.is.migration.client; resolution:=optional,
```

Comment:
Bad to add migration client as a dependency of identity core.

Reply:
Its set to resolution optional.

Reply:
Now you do not need this, I guess. with osgi.

Reply:
Yes. This is removed.